app/components/Button/Button.css

```diff
@@ -75,20 +85,60 @@ html[data-theme='dark'] .button {
   }
 }
 
-.small {
-  padding: 10px 15px;
-  font-size: 0.8rem;
+.danger {
+  background: none;
+  box-shadow: none;
+  border: 2px solid var(--color-red-2);
+  color: var(--color-red-2);
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    box-shadow: none;
+    border: 2px solid var(--color-red-2);
+    color: var(--color-almost-white-1);
+  }
+
+  &:disabled {
+    background: none;
+    color: var(--color-red-2);
+    opacity: 0.5;
+  }
 }
 
-.normal {
-  font-size: inherit;
+html[data-theme='dark'] .danger {
+  background: none;
+  box-shadow: none;
+  color: var(--color-red-2);
+
+  &:hover:not(:disabled) {
+    color: var(--color-dark-gray-5);
+  }
+
+  &:disabled {
+    color: var(--color-red-2);
+  }
 }
 
-.large {
-  font-size: 1.4rem;
+.success {
+  background: #2da44e;
```

Comment:
Should add these greens to the variables file

Reply:
Yes, I'm thinking of changing the variables up, as the current green colors are very ugly